jena-base/src/main/java/org/apache/jena/atlas/lib/StrUtils.java

```diff
@@ -85,17 +89,20 @@ public static int strCompareIgnoreCase(String s1, String s2) {
     }
 
     public static byte[] asUTF8bytes(String s) {
-        try { return s.getBytes("UTF-8") ; }
-        catch (UnsupportedEncodingException ex)
-        { throw new InternalErrorException("UTF-8 not supported!") ; } 
+        return s.getBytes(UTF_8) ; 
     }
 
     public static String fromUTF8bytes(byte[] bytes) {
-        try { return new String(bytes, "UTF-8") ; }
-        catch (UnsupportedEncodingException ex)
-        { throw new InternalErrorException("UTF-8 not supported!") ; } 
+        return new String(bytes, UTF_8) ; 
     }
     
+    /**
+     * @param x
+     * @return &lt;null&gt; if x == null, otherwise, x.toString()
+     * 
+     * @deprecated prefer {@link Objects#toString(Object, String)} with "&lt;null&gt;" or just {@link Objects#toString(Object)} 
+     */
+    @Deprecated
     public static String str(Object x) {
```

Comment:
I suggest we leave this, and not `@Deprecated`.  `str()` is supposed to be reminiscent of SPARQL `STR`.  `str(...)` is used elsewhere so this is a pattern.